parquet-arrow/src/main/java/org/apache/parquet/arrow/schema/SchemaConverter.java

```diff
@@ -245,7 +247,14 @@ public TypeMapping visit(Date type) {
 
       @Override
       public TypeMapping visit(Time type) {
```

Comment:
Does Timestamp Arrow type have the same problem? The visit method always maps to `primitive(INT64, TIMESTAMP_MILLIS)` regardless off Arrow type time unit.

Reply:
@rdblue @xhochy Colud you please check so far? I will fix Timestamp issue as an another pull request.